docs: copy edit gateway deployment page (#8399)
open-telemetry/opentelemetry.io: content/en/docs/collector/deployment/gateway/index.md

}
```

### Load-balancing exporter

For a concrete example of the centralized Collector deployment pattern, first
look at the load-balancing exporter. It has two main
configuration fields:

- The `resolver`, which determines where to find the downstream Collectors (or

>   backends). If you use the `static` subkey here, you must manually

Reviewer: Let's make this a complete sentence.

```suggestion
- The `resolver` determines where to find the downstream Collectors or
  backends. If you use the `static` subkey here, you must manually
```

  enumerate the Collector URLs. The other supported resolver is the DNS resolver,
  which periodically checks for updates and resolves IP addresses. For this
  resolver type, the `hostname` subkey specifies the hostname to query to
  obtain the list of IP addresses.
- Use the `routing_key` field to route

>   spans to specific downstream Collectors. If you set this field to `traceID`

Reviewer: ```suggestion
- The `routing_key` field routes
  spans to specific downstream Collectors. If you set this field to `traceID`
```

>   (default), the load-balancing exporter exports spans based on their

Reviewer: I'm not sure that `traceID` is still the default value for `routing_key`. At least, I couldn't find confirmation in the README. @open-telemetry/collector-contrib-approvers, could anyone confirm one way or the other?
 
```suggestion
- The `routing_key` field routes
  spans to specific downstream Collectors. If you set this field to `traceID`, the load-balancing exporter exports spans based on their
```

  `traceID`. Otherwise, if you use `service` for `routing_key`, it
  exports spans based on their service name, which is useful when using
  connectors like the [span metrics connector][spanmetrics-connector], so all
  spans of a service are sent to the same downstream Collector for metric

>   collection, guaranteeing accurate aggregations.

Reviewer: This sentence is a little long too.

```suggestion
  `traceID`. Otherwise, if you use `service` for `routing_key`, it
  exports spans based on their service name. This routing is useful when using
  connectors like the [span metrics connector][spanmetrics-connector] because all
  spans of a service are sent to the same downstream Collector for metric
  collection, guaranteeing accurate aggregations.
```

The load-balancing exporter emits metrics including
`otelcol_loadbalancer_num_backends` and `otelcol_loadbalancer_backend_latency`
that you can use to monitor the health and performance of the Collector serving
the OTLP endpoint.

## Combined deployment of Collectors as agents and gateways

Often, a deployment of multiple OpenTelemetry Collectors runs both
Collectors as gateways and as [agents](/docs/collector/deployment/agent/).

The following diagram shows an architecture for such a combined deployment:

- Use Collectors running in the agent deployment pattern (running on each
  host, similar to Kubernetes DaemonSets) to collect telemetry from services

>   running on the host and host telemetry, such as host metrics and scraped logs.

Reviewer: ```suggestion
  host, similar to Kubernetes DaemonSets) to collect telemetry from services
  running on the host as well as the host's own telemetry, such as host metrics and scraped logs.
```

- Use Collectors running in the gateway deployment pattern to process data,
  such as filtering, sampling, and exporting to backends.

![gateway](otel-gateway-arch.svg)

This combined deployment pattern is necessary when you use components in your
Collector that either must be unique per host or consume information
that is available only on the same host where the application runs:

- Receivers like the
  [`hostmetricsreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver)
  or
  [`filelogreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)
  must be unique per host instance. Running multiple instances of these

>   receivers results in duplicate data.

Reviewer: ```suggestion
  receivers on the same host results in duplicate data.
```
